[2024-06-03T22:35:07.408+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-03T22:35:07.426+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2022-03-01T00:00:00+00:00 [queued]>
[2024-06-03T22:35:07.433+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2022-03-01T00:00:00+00:00 [queued]>
[2024-06-03T22:35:07.434+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-03T22:35:07.445+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2022-03-01 00:00:00+00:00
[2024-06-03T22:35:07.453+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=31491) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-03T22:35:07.454+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2022-03-01T00:00:00+00:00', '--job-id', '141', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmp03_fnkhe']
[2024-06-03T22:35:07.455+0000] {standard_task_runner.py:63} INFO - Started process 31501 to run task
[2024-06-03T22:35:07.455+0000] {standard_task_runner.py:91} INFO - Job 141: Subtask blunders_etl
[2024-06-03T22:35:07.494+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2022-03-01T00:00:00+00:00 [running]> on host f55fff24b127
[2024-06-03T22:35:07.564+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2022-03-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-03-01T00:00:00+00:00'
[2024-06-03T22:35:07.565+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-03T22:35:07.566+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-03T22:35:07.572+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-06-03T22:45:52.802+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-03T22:45:52.803+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-03T22:45:52.811+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2022-03-01T00:00:00+00:00, execution_date=20220301T000000, start_date=20240603T223507, end_date=20240603T224552
[2024-06-03T22:45:52.848+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-03T22:45:52.859+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-03T22:45:52.861+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-08T20:00:22.364+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-08T20:00:22.386+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2022-03-01T00:00:00+00:00 [queued]>
[2024-06-08T20:00:22.394+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2022-03-01T00:00:00+00:00 [queued]>
[2024-06-08T20:00:22.394+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-08T20:00:22.407+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2022-03-01 00:00:00+00:00
[2024-06-08T20:00:22.415+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=31929) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-08T20:00:22.417+0000] {standard_task_runner.py:63} INFO - Started process 31938 to run task
[2024-06-08T20:00:22.417+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2022-03-01T00:00:00+00:00', '--job-id', '317', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmp6vuofig9']
[2024-06-08T20:00:22.418+0000] {standard_task_runner.py:91} INFO - Job 317: Subtask blunders_etl
[2024-06-08T20:00:22.456+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2022-03-01T00:00:00+00:00 [running]> on host bf0ade909435
[2024-06-08T20:00:22.525+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2022-03-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-03-01T00:00:00+00:00'
[2024-06-08T20:00:22.526+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-08T20:00:22.528+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-08T20:00:22.536+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-06-08T20:12:42.417+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-08T20:12:42.418+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-08T20:12:42.428+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2022-03-01T00:00:00+00:00, execution_date=20220301T000000, start_date=20240608T200022, end_date=20240608T201242
[2024-06-08T20:12:42.472+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-08T20:12:42.490+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-08T20:12:42.491+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-07-08T01:58:45.867+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-08T01:58:45.883+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2022-03-01T00:00:00+00:00 [queued]>
[2024-07-08T01:58:45.889+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2022-03-01T00:00:00+00:00 [queued]>
[2024-07-08T01:58:45.889+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-08T01:58:45.899+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2022-03-01 00:00:00+00:00
[2024-07-08T01:58:45.906+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=32845) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-08T01:58:45.908+0000] {standard_task_runner.py:63} INFO - Started process 32847 to run task
[2024-07-08T01:58:45.908+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2022-03-01T00:00:00+00:00', '--job-id', '709', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmps_y9w9ov']
[2024-07-08T01:58:45.909+0000] {standard_task_runner.py:91} INFO - Job 709: Subtask blunders_etl
[2024-07-08T01:58:45.943+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2022-03-01T00:00:00+00:00 [running]> on host 6569f5155f0c
[2024-07-08T01:58:46.007+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2022-03-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-03-01T00:00:00+00:00'
[2024-07-08T01:58:46.008+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-08T01:58:46.009+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-07-08T01:58:46.015+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-07-08T02:09:14.926+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-08T02:09:14.927+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-08T02:09:14.934+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2022-03-01T00:00:00+00:00, execution_date=20220301T000000, start_date=20240708T015845, end_date=20240708T020914
[2024-07-08T02:09:14.956+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-08T02:09:14.976+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-08T02:09:14.977+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
